app/src/models/hash.ts

```diff
@@ -0,0 +1,5 @@
+type HashableType = number | string | boolean | undefined | null
+
+export function createHash(items: HashableType[]) {
```

Comment:
Thanks so much for doing this @sergiou87. Should prevent us from similar mistakes in the future. Two small thoughts. 1) Could we make the argument list dynamic to get rid of the `[]` brackets?


And second, could we add a little documentation to the method? We try to do so for all exported or otherwise public methods.

```suggestion
/**
  Creates a string representation of the provided arguments.
  
  This is a helper function used to create a string representation of
  an object based on its properties for the purposes of simple equality
  comparisons.
**/
export function createHash(...items: HashableType[]) {
```

Lastly, I wonder if we should call this method `createEqualityHash` or something to differentiate it from [createHash](https://nodejs.org/api/crypto.html#crypto_crypto_createhash_algorithm_options).